Makefile

```diff
@@ -79,6 +79,9 @@ client: generate
 ci-rp: fix-macos-vendor
 	docker build . -f Dockerfile.ci-rp --ulimit=nofile=4096:4096 --build-arg REGISTRY=$(REGISTRY) --build-arg VERSION=$(VERSION) --no-cache=$(NO_CACHE)
 
+ci-clean:
```

Comment:
since we are adding a new make target, we should add it to `.PHONY` at the bottom of the file.

Reply:
Done :)